msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/ManagedIdentityRetryPolicy.java

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.aad.msal4j;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Retry policy for most Managed Identity scenarios
+ */
+class ManagedIdentityRetryPolicy implements IRetryPolicy {
+    private static final int RETRY_NUM = 3;
+    private static final int RETRY_DELAY_MS = 1000;
```

Comment:
should these be static?

Reply:
`private final` means they are associated with an instance and would be initialized every time a RetryPolicy is created, `private static final` means they are (more-or-less) initialized once for the lifetime of the application.

It's a pretty minor boost in the grand scheme of things, but since they will never change and will be used by every instance of RetryPolicy there's no need to initialize them every time.